app/templates/records/new_to_archives.html

```diff
@@ -16,8 +17,114 @@
+          <h3 class="tna-accordion__heading tna-heading-s">
+            What are archives?
+          </h3>
+          <div class="tna-accordion__body" id="accordion-what-are-archives">
+            <ul class="tna-ul">
+              <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</li>
+              <li>Nulla dignissim vehicula magna, et hendrerit quam iaculis a. Mauris in ultricies enim.</li>
+              <li>Donec bibendum est leo, sed dapibus mauris facilisis vitae.</li>
+              <li>Quisque hendrerit condimentum nisl, non volutpat ex eleifend at. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Morbi id suscipit felis, sed tincidunt arcu. Etiam vel blandit diam, vitae commodo mi.</li>
+            </ul>
+          </div>
+        </div>
+        <div class="tna-accordion__item" data-isopen="false">
+          <h3 class="tna-accordion__heading tna-heading-s">
+            How to use archives?
+          </h3>
+          <div class="tna-accordion__body" id="accordion-how-to-use">
+            <ul class="tna-ul">
+              <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</li>
+              <li>Nulla dignissim vehicula magna, et hendrerit quam iaculis a. Mauris in ultricies enim.</li>
+              <li>Donec bibendum est leo, sed dapibus mauris facilisis vitae.</li>
+              <li>Quisque hendrerit condimentum nisl, non volutpat ex eleifend at. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Morbi id suscipit felis, sed tincidunt arcu. Etiam vel blandit diam, vitae commodo mi.</li>
+            </ul>
+          </div>
+        </div>
+        
+      </div>
+       <!--end accordion-->
+
+      <!--card/hero element-->
+      <div class="tna-hero tna-!--margin-top-m">
+        <figure class="tna-hero__figure">
+          <div class="tna-container tna-hero__inner">
+            <div class="tna-column tna-column--width-2-3 tna-column--full-small tna-column--full-tiny tna-hero__content">
+              <div class="tna-hero__content-inner tna-background-accent">
+                <hgroup class="tna-hgroup-l tna-hero__heading">
+                  
+                  <h3 class="tna-hgroup__title">
+                    Explore the collection
+                  </h3>
+                </hgroup>
+                <p>Discover the stories behind the records. Read articles about the incredible people whose lives can be traced in the archive, or go deep into a new perspectives on historical events.</p>
+                <a href="/explore-the-collection/" class="tna-button tna-!--margin-top-s">
+                  Explore
+                </a>
+              </div>
+
+            </div>
+          </div>
+          <picture class="tna-hero__image">
+            <img src="../../../static/assets/images/festival-hall.jpg" alt="An illustraion of the Festival of Britian" width="1200" height="480">
```

Comment:
```suggestion
            <img src="{{ static('assets/images/assets/images/festival-hall.jpg') }}" alt="An illustration of the Festival of Britain" width="1200" height="480">
```

Reply:
Code update to the following:
`<img src="{{ static('assets/images/festival-hall.jpg') }}" alt="An illustration of the Festival of Britain" width="1200" height="480">`